File: 3st/Hyeonseo/programmer/각도기.java
class Solution {
    public int solution(int angle) {
        int answer = 0;
        if(angle < 90) {
            answer = 1;
        }
        else {
            answer = 3;
            if(angle == 90) answer = 2;
            if(angle == 180) answer = 4;
        }
        return answer;
    }
}

File: 3st/Hyeonseo/programmer/각도기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int angle) {
    int answer = 1;
    if(angle == 90) {
        answer = 2;
    }
    else if(angle > 90 && angle < 180) {
        answer = 3;
    }
    else if(angle == 180) {
        answer = 4;
    }
    return answer;
}